react/components/ShippingSimulator/index.js

```diff
@@ -41,6 +41,12 @@ const ShippingSimulator = ({
     setIsValid(postalCode.valid)
   }
 
+  const keyPress = e => {
+    if(e.which == 13){
```

Comment:
This attribute is deprecated, use `e.key` (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values)